frontend/tests/components/opportunity/OpportunityDescription.test.tsx

```diff
@@ -59,7 +59,9 @@ describe("OpportunityDescription", () => {
     expect(descriptionHeading).toBeInTheDocument();
 
     const sanitizedSummaryDescription = DOMPurify.sanitize(
-      mockOpportunityData.summary.summary_description,
+      mockOpportunityData.summary.summary_description
```

Comment:
why do we have to do a check here (and elsewhere in the file), aren't we supplying the mock data directly to ensure that we we don't pass incomplete data in these cases?

Reply:
@doug-s-nava the sanitize function won't take a null value.

Reply:
Looking more closely I'm a little confused about what we're testing here, would love to have a quick call to straighten it out

Reply:
I updated this, thanks.